dev/preview/infrastructure/harvester/dns.tf

```diff
@@ -29,7 +29,7 @@ resource "google_dns_record_set" "root-wc" {
 resource "google_dns_record_set" "root-wc-ws" {
   provider = google
 
-  name = "*.ws.${var.preview_name}.${data.google_dns_managed_zone.preview-gitpod-dev.dns_name}"
+  name = "*.ws-dev.${var.preview_name}.${data.google_dns_managed_zone.preview-gitpod-dev.dns_name}"
```

Comment:
Missed those 2.

Reply:
I will add, but this doesn't do much for mitigation, because these terraforms will only be executed if they are newly created or run with annotation `with-clean-slate-deployment=true`, but other changes to the installer in the `.werft` folder will be triggered immediately on the next build, so The only way to do this is to rebuild the preview environment again with `with-clean-slate-deployment=true`

Reply:
i.e. When this PR is merged, any preview environment will be installed with `ws-dev`, whether new or already existing, but the already existing ones will not change the cert and dns unless they are re-run with annotation `with-clean-slate-deployment=true`